### PR TITLE
Commerce: Updated articles for v14

### PR DESCRIPTION
**Pull request by eshanrnh**
## Description

Some changes related to v14

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [x] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

Commerce v14

## Deadline (if relevant)

Anytime

### CODE REVIEW

File: 14/umbraco-commerce/installation/install.md [hunk 1]
```diff
@@ -5,18 +5,18 @@ description: >-
 
 # Installation
 
-Learn how to install Umbraco Commerce into your Umbraco CMS implementation.
+In this article, you can learn how to install Umbraco Commerce into your Umbraco CMS implementation.
```
Comment (sofietoft) [inline 14/umbraco-commerce/installation/install.md:8]:
```suggestion
In this article, you will learn how to install Umbraco Commerce into your Umbraco CMS implementation.
```

File: 14/umbraco-commerce/installation/install.md [hunk 2]
```diff
@@ -5,18 +5,18 @@ description: >-
...
 
-You can also find information about how to upgrade and how to install and activate your Umbraco Commerce license.
+You can also find information about how to upgrade, install, and activate your Umbraco Commerce license.
 
 ## NuGet Package Installation
 
 Umbraco Commerce is available via [NuGet.Org](https://www.nuget.org/packages/Umbraco.Commerce/).
 
-To install Umbraco Commerce via NuGet you can run the following command directly in the NuGet Manager Console window:
+To install Umbraco Commerce via NuGet, you can run the following command directly in the NuGet Manager Console window:
```
Comment (sofietoft) [inline 14/umbraco-commerce/installation/install.md:16]:
```suggestion
To install Umbraco Commerce via NuGet, run the following command directly in the NuGet Manager Console window:
```

File: 14/umbraco-commerce/installation/install.md [hunk 3]
```diff
@@ -96,6 +96,10 @@ For most sites using a single solution, the above will be all you need to instal
    </tbody>
 </table>
 
+{% hint style="info" %}
+If you encounter an SQLite error after installing Umbraco Commerce, you may need to enable SQLite support. For more information, see the [Configure SQLite support](https://docs.umbraco.com/umbraco-commerce/how-to-guides/configure-sqlite-support) article.
```
Comment (sofietoft) [inline 14/umbraco-commerce/installation/install.md:100]: Wouldn't it be better to use a relate link instead? So, something like `../../installation/install` instead.

File: 14/umbraco-commerce/reference/shipping/dynamic-rate-shipping.md [hunk 4]
```diff
@@ -8,35 +8,35 @@ Dynamic rate shipping in Umbraco Commerce allows you to define a series of range
...
-* Choose the range unit to base the rates upon
-* Click `Add Range` to define each range
+6. Choose the range unit to base the rates upon.
+7. Click **Add Range** to define each range.
 
+    ![Shipping Method Rates](../../media/v14/shipping-method-dynamic-rates.png)
 
-![Shipping Method Rates](../../media/v14/shipping-method-dynamic-rates.png)
+8. Enter the **From** and **To** value of the range.
+9. Enter the rate details from the available rate options leaving blank any option you don't wish to apply.
```
Comment (sofietoft) [inline 14/umbraco-commerce/reference/shipping/dynamic-rate-shipping.md:34]:
```suggestion
9. Enter the rate details from the available rate options leaving blank any option you do not wish to apply.
```

**Review by sofietoft — COMMENTED** [review]
Looks great!
I've added a couple of suggestions 💪

File: 14/umbraco-commerce/installation/install.md (hunk 3 above)
Comment (eshanrnh) [reply 14/umbraco-commerce/installation/install.md]: Done

**Review by sofietoft — APPROVED** [review]
LGTM 💪

**Pull request merged**